app/src/main/java/fr/free/nrw/commons/upload/UploadModel.java

```diff
@@ -142,6 +142,9 @@ private UploadItem getUploadItem(UploadableFile uploadableFile,
                 createdTimestampSource);
         if (place != null) {
             uploadItem.title.setTitleText(place.name);
+            if(uploadItem.descriptions.isEmpty()) {
```

Comment:
Should we rather just not update the descriptions if they are empty, instead of creating a new object ?

Reply:
And another question can be is it normal that descriptions are empty at this state?